index.js

```diff
@@ -196,7 +196,13 @@ Polyglot.prototype.t = function (key, options) {
   }
   if (typeof phrase === 'string') {
     opts = assign({}, opts);
-    result = choosePluralForm(phrase, this.currentLocale, opts.smart_count);
+    if (typeof opts.smart_count === 'object' && isValidSmartObject(opts.smart_count)) {
```

Comment:
this typeof check should be moved into `isValidSmartObject`, since `isValidSmartObject(null)` should be able to return false.